src/test/java/phrasecount/PhraseCounterTest.java

```diff
@@ -15,6 +15,7 @@
 import io.fluo.api.mini.MiniFluo;
 import io.fluo.api.types.TypedSnapshot;
 import io.fluo.api.types.TypedSnapshotBase.Value;
+import io.fluo.core.util.PortUtils;
```

Comment:
Thinking we should avoid using non API code in examples.  Would not want to encourage users to use this code in their test.